### PR TITLE
Add io.vikunja.Desktop

### PR DESCRIPTION
**Pull request by kolaente**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub.
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

Resolves https://github.com/go-vikunja/desktop/issues/7

### CODE REVIEW

**Comment by kolaente** [comment]
Upstream references:

https://kolaente.dev/vikunja/api/issues/1572
https://github.com/go-vikunja/desktop/issues/7
https://codeberg.org/RaptaG/vikunja-flathub-prepare/src/branch/main

**Review by RaptaG — APPROVED** [review]

File: flathub.json
Comment (hfiguiere) [inline flathub.json:2]: this is disappointing

File: io.vikunja.Desktop.appdata.xml
Comment (hfiguiere) [inline io.vikunja.Desktop.appdata.xml:16]: missing `screenshots` and a proper `description`

File: io.vikunja.Desktop.appdata.xml
Comment (hfiguiere) [inline io.vikunja.Desktop.appdata.xml:8]: this is an invalid launchable. it should be the name of a .desktop file.

File: io.vikunja.Desktop.appdata.xml
Comment (hfiguiere) [inline io.vikunja.Desktop.appdata.xml:3]: please don't use `Desktop` and the last component. It is confusing and non descript.

File: io.vikunja.Desktop.yml
Comment (hfiguiere) [inline io.vikunja.Desktop.yml:3]: neither are valid.
```suggestion
```

File: io.vikunja.Desktop.yml
Comment (hfiguiere) [inline io.vikunja.Desktop.yml:21]: does it really support wayland out of the box?

Comment (hfiguiere) [reply io.vikunja.Desktop.yml:21]: Also `--share=ipc` is necessary.
and I suspect `--device=dri`

File: io.vikunja.Desktop.yml
Comment (hfiguiere) [inline io.vikunja.Desktop.yml:23]: do you really need this?

File: flathub.json
Comment (kolaente) [reply flathub.json:2]: Wanna do a PR at our repo to enable multi-arch builds?

File: io.vikunja.Desktop.yml
Comment (hfiguiere) [inline io.vikunja.Desktop.yml:24]:
```suggestion
```

You don't need that.

File: io.vikunja.Desktop.appdata.xml
Comment (kolaente) [reply io.vikunja.Desktop.appdata.xml:16]: Is there any reference on how I should provide these?

File: io.vikunja.Desktop.appdata.xml
Comment (kolaente) [reply io.vikunja.Desktop.appdata.xml:8]: Ah, wasn't sure about that. Will fix.

File: io.vikunja.Desktop.appdata.xml
Comment (hfiguiere) [inline io.vikunja.Desktop.appdata.xml:5]: "all platforms" is a bold claim when you exclude aarch64.

File: io.vikunja.Desktop.appdata.xml
Comment (kolaente) [reply io.vikunja.Desktop.appdata.xml:8]: Fixed.

File: io.vikunja.Desktop.appdata.xml
Comment (hfiguiere) [inline io.vikunja.Desktop.appdata.xml:6]: this is not a valid SPDX license identifier.

File: io.vikunja.Desktop.appdata.xml
Comment (hfiguiere) [inline io.vikunja.Desktop.appdata.xml:1]: this file is never installed. 

as it been submitted for inclusion upstream?

File: io.vikunja.Desktop.yml
Comment (kolaente) [reply io.vikunja.Desktop.yml:3]: So I should remove them?

File: io.vikunja.Desktop.yml
Comment (kolaente) [reply io.vikunja.Desktop.yml:21]: It's an electron app, AFAIK it does.

File: io.vikunja.Desktop.yml
Comment (hfiguiere) [inline io.vikunja.Desktop.yml:34]: I don't see an icon or a .desktop file installed.

File: io.vikunja.Desktop.yml
Comment (hfiguiere) [inline io.vikunja.Desktop.yml:37]: neither of these will work. this is not a valid sha256.

File: io.vikunja.Desktop.appdata.xml
Comment (kolaente) [reply io.vikunja.Desktop.appdata.xml:6]: Fixed

File: io.vikunja.Desktop.appdata.xml
Comment (hfiguiere) [reply io.vikunja.Desktop.appdata.xml:3]: `Vikunja` would be appropriate.

File: io.vikunja.Desktop.appdata.xml
Comment (hfiguiere) [reply io.vikunja.Desktop.appdata.xml:16]: From the documentatin you acknowledged having read. https://docs.flathub.org/docs/for-app-authors/appdata-guidelines/

File: io.vikunja.Desktop.yml
Comment (hfiguiere) [reply io.vikunja.Desktop.yml:21]: afaik electron usually doesn't.

File: io.vikunja.Desktop.appdata.xml
Comment (kolaente) [reply io.vikunja.Desktop.appdata.xml:1]: So it should be just included in the tarball?

File: io.vikunja.Desktop.yml
Comment (kolaente) [reply io.vikunja.Desktop.yml:37]: I thought this would get expanded from the variables defined at the beginning of this file?

File: io.vikunja.Desktop.yml
Comment (kolaente) [reply io.vikunja.Desktop.yml:33]: They are in the tarball.

File: io.vikunja.Desktop.yml
Comment (hfiguiere) [reply io.vikunja.Desktop.yml:37]: where in the dcoumentation does it say that? (so I can actually fix the documentation)

File: io.vikunja.Desktop.appdata.xml
Comment (hfiguiere) [reply io.vikunja.Desktop.appdata.xml:1]: the prefered way is that it be maintained upstream and shipped with the application. It's not specific to flatpak at all.

File: io.vikunja.Desktop.yml
Comment (hfiguiere) [reply io.vikunja.Desktop.yml:33]: then they are not installed in the right places.

File: io.vikunja.Desktop.appdata.xml
Comment (RaptaG) [reply io.vikunja.Desktop.appdata.xml]: Well this could be my bad though I'm sure I included `.desktop` in the original

File: io.vikunja.Desktop.yml
Comment (RaptaG) [reply io.vikunja.Desktop.yml]: How so?

File: io.vikunja.Desktop.yml
Comment (RaptaG) [reply io.vikunja.Desktop.yml]: Since there is an x11 fallback (which I added), it's fine

File: io.vikunja.Desktop.yml
Comment (RaptaG) [reply io.vikunja.Desktop.yml:19]: All of them?
Notifications yeah

File: io.vikunja.Desktop.yml
Comment (RaptaG) [reply io.vikunja.Desktop.yml]: I run the hash check (`sha256sum Vikunja\ Desktop-0.21.0.tar.gz`) locally directly after downloading the file (https://dl.vikunja.io/desktop/0.21.0/Vikunja%20Desktop-0.21.0.tar.gz) and that was the hash I got

Comment (kolaente) [reply io.vikunja.Desktop.yml]: And the variable interpolation?

**Comment by RaptaG** [comment]
Hello and sorry I made some outdated comments, I had not refreshed the tab

File: io.vikunja.Desktop.yml
Comment (hfiguiere) [reply io.vikunja.Desktop.yml]: no it's not. 

the fallback is for the case when the host doesn't have wayland.

File: io.vikunja.Desktop.yml
Comment (hfiguiere) [reply io.vikunja.Desktop.yml:19]: line 23 is the subject.

File: io.vikunja.Desktop.yml
Comment (hfiguiere) [inline io.vikunja.Desktop.yml:1]: the file name MUST match the id.

File: io.vikunja.Desktop.appdata.xml
Comment (RaptaG) [reply io.vikunja.Desktop.appdata.xml]: @kolaente you could name it `io.vikunja.App`

File: io.vikunja.Desktop.yml
Comment (hfiguiere) [reply io.vikunja.Desktop.yml:30]: This is still unresolved (but you resolved the thread). Its not a "nice to have".

File: io.vikunja.Desktop.appdata.xml
Comment (hfiguiere) [reply io.vikunja.Desktop.appdata.xml]: this is differently bad taste. At least it's not `Desktop`.

File: io.vikunja.Desktop.appdata.xml
Comment (kolaente) [reply io.vikunja.Desktop.appdata.xml:16]: Well the links I acknowledged didn't exactly link back to the article you mentioned. But that's nitpicking.

File: io.vikunja.Desktop.yml
Comment (RaptaG) [reply io.vikunja.Desktop.yml]: @hfiguiere

File: io.vikunja.Desktop.yml
Comment (kolaente) [reply io.vikunja.Desktop.yml]: removed

File: io.vikunja.Desktop.yml
Comment (kolaente) [reply io.vikunja.Desktop.yml]: @RaptaG what do you mean?

File: io.vikunja.Vikunja.appdata.xml
Comment (hfiguiere) [inline io.vikunja.Vikunja.appdata.xml:8]: this too is incorrect.

File: io.vikunja.Vikunja.yml
Comment (hfiguiere) [inline io.vikunja.Vikunja.yml:17]: `share=ipc` is necessary.

File: io.vikunja.Vikunja.yml
Comment (hfiguiere) [inline io.vikunja.Vikunja.yml:20]: third time I ask. Remove this.

```suggestion
```

File: io.vikunja.Vikunja.yml
Comment (hfiguiere) [inline io.vikunja.Vikunja.yml:30]: I still so no .desktop. appstream or icon installed in the right locations.

File: io.vikunja.Desktop.yml
Comment (RaptaG) [reply io.vikunja.Desktop.yml]: Doesn't Vikunja notify you when a tasks time is up?

File: io.vikunja.Vikunja.yml
Comment (hfiguiere) [reply io.vikunja.Vikunja.yml:17]: and I suspect `device=dri` is needed too.

File: io.vikunja.Desktop.yml
Comment (RaptaG) [reply io.vikunja.Desktop.yml]: Ah it's via e-mail nvm

File: io.vikunja.Desktop.appdata.xml
Comment (kolaente) [reply io.vikunja.Desktop.appdata.xml:1]: Looks like Electron builder actually does this, so the file here is redundant. I've removed it.

Comment (RaptaG) [reply io.vikunja.Desktop.appdata.xml:1]: I can PR it @kolaente

File: flathub.json
Comment (hfiguiere) [inline flathub.json:3]: why do you need this? There is no f-e-d-c setup.

File: io.vikunja.Vikunja.yml
Comment (hfiguiere) [inline io.vikunja.Vikunja.yml:29]:
```suggestion
      - install -D vikunja.sh "${FLATPAK_DEST}/bin/vikunja"
```

File: io.vikunja.Desktop.appdata.xml
Comment (RaptaG) [reply io.vikunja.Desktop.appdata.xml:1]: Done https://kolaente.dev/vikunja/desktop/pulls/162

File: io.vikunja.Desktop.yml
Comment (kolaente) [reply io.vikunja.Desktop.yml]: I mean the `{{ version-hash }}` instead of putting the hash there directly. There was a variable `version-hash` at the top level of the yml, I assume that was meant to interpolate the usage in `{{`? Where did you get that from?

Comment (RaptaG) [reply io.vikunja.Desktop.yml]: For accessibility, I set the hash and the app version to 2 separate variables at the start of the file so that they can be easier and better updated when an update arrives

Comment (kolaente) [reply io.vikunja.Desktop.yml]: I like the idea, but it looks like that won't work because flatpack does not support expanding variables.

Comment (barthalion) [reply io.vikunja.Desktop.yml]: Yeah, wherever you picked it from, it's not in YAML spec. You could use anchors for that, but there's no interpolation.

**Comment by barthalion** [comment]
bot, build io.vikunja.Vikunja

**Comment by RaptaG** [comment]
@kolaente has told me to carry on this, so you may close this PR I will open a new one soon

**Comment by kolaente** [comment]
@RaptaG I've added you to my fork repo, you should have write permissions now and should be able to push to this branch.

**Comment by hfiguiere** [comment]
Build failure:
```
Error: Command 'vikunja-desktop' not found
```

File: io.vikunja.Vikunja.yml
Comment (hfiguiere) [inline io.vikunja.Vikunja.yml:14]: this is inconsistent with line 28.

Comment (kolaente) [reply io.vikunja.Vikunja.yml]: Fixed

**Comment by hfiguiere** [comment]
bot, build io.vikunja.Vikunja

File: io.vikunja.Vikunja.yml
Comment (hfiguiere) [reply io.vikunja.Vikunja.yml:29]: This is the reason the build fails right now. I have lost count of how many times I said this is not done.  But there is something but trying the same thing over and over and expecting a different result. It won't work.

File: io.vikunja.Vikunja.yml
Comment (RaptaG) [reply io.vikunja.Vikunja.yml:17]: > and I suspect `device=dri` is needed too.

What does it do?

File: io.vikunja.Vikunja.yml
Comment (RaptaG) [reply io.vikunja.Vikunja.yml:29]: I will fix it now

**Comment by RaptaG** [comment]
Which location would be ideal to place the `.desktop` and the icons into @hfiguiere ?

**Comment by RaptaG** [comment]
@hfiguiere I made a commit covering most (if not all) your suggestions.

Now I need some help:

1.  Is [this](https://github.com/kolaente/flathub/blob/c9d7d35f1a997248d4bf22d1c55dfc1fe2161fe0/io.vikunja.Vikunja.yml#L28), [this](https://github.com/kolaente/flathub/blob/c9d7d35f1a997248d4bf22d1c55dfc1fe2161fe0/io.vikunja.Vikunja.yml#L30) and [that](https://github.com/kolaente/flathub/blob/c9d7d35f1a997248d4bf22d1c55dfc1fe2161fe0/vikunja.sh#L3) correct?
I changed those according to the [original template](https://gitlab.parmentier.io/lparmentier/flatpak-vikunja), but I'm unsure on how to properly build the package, so I'd love some feedback on how to do it the right way.
2. [I made a `.desktop` file](https://kolaente.dev/RaptaG/desktop/src/branch/main/io.vikunja.Vikunja.desktop), but I don't know how to set it to access the icon's location (based on [the current one](https://github.com/kolaente/flathub/blob/c9d7d35f1a997248d4bf22d1c55dfc1fe2161fe0/io.vikunja.Vikunja.yml#L26)). Can I have some help?

File: flathub.json
Comment (hfiguiere) [inline flathub.json:1]: this remain disappointing. I thought the goal for using bloat like Electron was to support many platform and you just do the opposite.

File: io.vikunja.Vikunja.yml
Comment (hfiguiere) [inline io.vikunja.Vikunja.yml:30]: no. that will install in `/app/app` That is not what was asked.

File: flathub.json
Comment (RaptaG) [reply flathub.json:1]: 1. I have suggested the use of Tauri and [there is a PR for that](https://kolaente.dev/vikunja/desktop/pulls/127).
2. I am not the maintainer of the project
3. Please be more polite. All of your comments have an aggressive tone which can be both offending and can be discouraging.

File: io.vikunja.Vikunja.yml
Comment (RaptaG) [reply io.vikunja.Vikunja.yml:30]: Can you show me how to do it the right way?

File: io.vikunja.Vikunja.yml
Comment (RaptaG) [reply io.vikunja.Vikunja.yml:26]: So I have to include the files in the repo right?

File: vikunja.sh
Comment (RaptaG) [reply vikunja.sh:3]: What should I do?

File: flathub.json
Comment (hfiguiere) [reply flathub.json:1]: @RaptaG well I got that answer initially https://github.com/flathub/flathub/pull/4319#discussion_r1258589072

Electron DO support `aarch64`.

**Comment by RaptaG** [comment]
@hfiguiere I did what you asked.

I can include the appdata, icon and desktop file inside the Flathub repo and if yes where?
They are not bundled inside the `tar.gz`

**Comment by barthalion** [comment]
Yes, you need to at least include them in the repo. They should be moved to Vikunja repo eventually.

**Comment by RaptaG** [comment]
> Yes, you need to at least include them in the repo. They should be moved to Vikunja repo eventually..

Yes, this can be done once it has been ported over to Tauri..

**Comment by RaptaG** [comment]
bot, build io.vikunja.Vikunja

**Comment by RaptaG** [comment]
> Build [52723 failed](https://buildbot.flathub.org/#/builders/6/builds/52723)

https://buildbot.flathub.org/#/builders/35/builds/6512/steps/8/logs/stdio

Can anyone help me debug the issue, I don't see the clear root of the problem..

**Comment by RaptaG** [comment]
@barthalion you helped with the previous failure, could you enlighten me this time too?

**Comment by hfiguiere** [comment]
```
io.vikunja.Vikunja:                AppData problem: tag-missing : <description> required
```

**Comment by RaptaG** [comment]
bot, build io.vikunja.Vikunja

**Comment by RaptaG** [comment]
It failed again, can I have the error @hfiguiere

**Comment by RaptaG** [comment]
bot, build io.vikunja.Vikunja

**Comment by RaptaG** [comment]
bot, cancel build io.vikunja.Vikunja

**Comment by RaptaG** [comment]
bot, cancelbuild io.vikunja.Vikunja

**Comment by RaptaG** [comment]
bot, build io.vikunja.Vikunja

**Comment by RaptaG** [comment]
The building process now works!

## **PLEASE DO NOT MERGE YET AS SOME ADJUSTMENTS ARE YET TO BE MADE**

@kolaente I get some errors on the console running the package:

```bash
raptag@laptop:~/Git/flathub
$ flatpak run io.vikunja.Vikunja
Gtk-Message: 18:16:36.460: Failed to load module "canberra-gtk-module"
[2:0712/181636.707755:ERROR:bus.cc(399)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[2:0712/181636.708026:ERROR:bus.cc(399)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[2:0712/181636.721862:ERROR:bus.cc(399)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[41:0712/181636.783432:ERROR:egl_util.cc(44)] Failed to load GLES library: /app/bin/extra/libGLESv2.so: /app/bin/extra/libGLESv2.so: cannot open shared object file: No such file or directory
[41:0712/181636.803191:ERROR:viz_main_impl.cc(186)] Exiting GPU process due to errors during initialization
Server started on port 45735
[68:0712/181637.353350:ERROR:egl_util.cc(44)] Failed to load GLES library: /app/bin/extra/libGLESv2.so: /app/bin/extra/libGLESv2.so: cannot open shared object file: No such file or directory
[68:0712/181637.408616:ERROR:viz_main_impl.cc(186)] Exiting GPU process due to errors during initialization
[85:0712/181637.537591:ERROR:egl_util.cc(44)] Failed to load GLES library: /app/bin/extra/libGLESv2.so: /app/bin/extra/libGLESv2.so: cannot open shared object file: No such file or directory
[85:0712/181637.565086:ERROR:viz_main_impl.cc(186)] Exiting GPU process due to errors during initialization
```

This how the startup screen looks:
image


So what needs to be fixed:

- [ ] Icon not being displayed
- [ ] The error on the app
- [ ] The errors on the console

As always, feedback is appreciated!

**Comment by kolaente** [comment]
@RaptaG The error in Vikunja itself is kind of expected as you need an api to connect to. It should not be displayed as an error but that's unrelated to the electron wrapper or flatpack.
If you enter an api URL (try.vikunja.io should work) can you use the app as expected?

Thanks a lot for your efforts so far!

**Comment by RaptaG** [comment]
@kolaente what about the console logs?

**Comment by RaptaG** [comment]
Also, is the icon issue somewhat related to https://kolaente.dev/vikunja/desktop/commit/a92eb5c70ee17e153cc8a8f0381b62f99995a565

**Comment by RaptaG** [comment]
Other than that, the app works as intended

**Comment by RaptaG** [comment]
The log now outputs an Electron error which seems harmless and does not affect Vikunja: https://github.com/intel/libva/issues/518

**Comment by RaptaG** [comment]
bot, build io.vikunja.Vikunja

**Comment by RaptaG** [comment]
Now I have to take some screenshots

**Comment by RaptaG** [comment]
bot, build io.vikunja.Vikunja

**Comment by RaptaG** [comment]
bot, build io.vikunja.Vikunja

**Comment by RaptaG** [comment]
@hfiguiere can you help me find out why it failed again?

I did everything I was asked to.

**Everything** is valid.
**Nothing** is missing.

File: io.vikunja.Vikunja.appdata.xml
Comment (RaptaG) [reply io.vikunja.Vikunja.appdata.xml:36]: Ah right I forgot that

**Comment by RaptaG** [comment]
bot, build io.vikunja.Vikunja

**Comment by RaptaG** [comment]
I think the PR is now ready for merge.

@kolaente I need your approval too

**Comment by kolaente** [comment]
> The log now outputs an Intel-affecting Electron error which seems harmless and does not affect Vikunja: https://github.com/intel/libva/issues/518

Then let's leave it as is.

File: io.vikunja.Vikunja.appdata.xml
Comment (kolaente) [inline io.vikunja.Vikunja.appdata.xml:27]: Not quite sure about hotlinking but I think we should use https://vikunja.io/images/vikunja/01-task-overview.png here as it's more up to date

**Review by kolaente — COMMENTED** [review]
Two small nits regarding the screenshots, LGTM otherwise!

**Comment by RaptaG** [comment]
@barthalion @hfiguiere You may merge this now, unless you have any other change to suggest.

**Review by RaptaG — APPROVED** [review]

**Comment by hfiguiere** [comment]
bot, build io.vikunja.Vikunja

**Comment by barthalion** [comment]
It seems to be unhappy about something, as it claims `You are offline.`.

**Comment by RaptaG** [comment]
> It seems to be unhappy about something, as it claims `You are offline.`.

Where? Is there a network permission I need to add?

**Comment by barthalion** [comment]
Does it work for you locally?

**Comment by RaptaG** [comment]
> Does it work for you locally?

yes

**Comment by barthalion** [comment]
Well, it doesn't for me, and there's nothing in the logs.

```
[bpiotrowski@gnomuru ~]$ flatpak run io.vikunja.Vikunja

[2:0719/084159.013731:ERROR:bus.cc(399)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
Gtk-Message: 08:41:59.224: Failed to load module "canberra-gtk-module"
Gtk-Message: 08:41:59.227: Failed to load module "canberra-gtk-module"
[2:0719/084159.239113:ERROR:bus.cc(399)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[2:0719/084159.239140:ERROR:bus.cc(399)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
Server started on port 45735
```

**Comment by RaptaG** [comment]
> Well, it doesn't for me, and there's nothing in the logs.
> 
> ```
> [bpiotrowski@gnomuru ~]$ flatpak run io.vikunja.Vikunja
> 
> [2:0719/084159.013731:ERROR:bus.cc(399)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
> Gtk-Message: 08:41:59.224: Failed to load module "canberra-gtk-module"
> Gtk-Message: 08:41:59.227: Failed to load module "canberra-gtk-module"
> [2:0719/084159.239113:ERROR:bus.cc(399)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
> [2:0719/084159.239140:ERROR:bus.cc(399)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
> Server started on port 45735
> ```

Can you force-disable wayland and try again @barthalion

**Comment by kolaente** [comment]
@barthalion Does it work if you download a build from [here](https://dl.vikunja.io/desktop/unstable) and running it directly without Flatpack?

Just trying to figure out if its a general problem with Vikunja or a problem with the packaging.

**Comment by hfiguiere** [comment]
Doesn't any kind of socket (incl. unix domain) require `network` permission?

**Comment by honnip** [comment]
Network permission is required as it needs to connect to the server.  
[little pr](https://github.com/kolaente/flathub/pull/1) for this

**Comment by RaptaG** [comment]
> Network permission is required as it needs to connect to the server.  
> [little pr](https://github.com/kolaente/flathub/pull/1) for this

Will merge it after you answer my comment :P

**Comment by honnip** [comment]
I'm not sure what comment you're talking about

**Comment by RaptaG** [comment]
> I'm not sure what comment you're talking about


https://github.com/kolaente/flathub/pull/1/files#diff-7439282b961560b342143698a4b6016e4b3e981eb643b1d0b96ca07abf1d2f5aR9

File: io.vikunja.Vikunja.desktop
Comment (hfiguiere) [inline io.vikunja.Vikunja.desktop:9]:
```suggestion
Exec=vikunja
```

this always match the `command` in the manifest (more or less arguments)

Comment (honnip) [reply io.vikunja.Vikunja.desktop]: Fixed

**Pull request closed without merging**
